src/riot/Lesson/LessonContent.riot.html

```diff
@@ -1,11 +1,15 @@
 <LessonContent>
     <LessonFrame
         if="{ props.lesson.ready && state.cardNumber <= state.cards.length }"
-        extraStyleClasses="content"
+        extraStyleClasses="{state.card.tag.includes('Choice') ? 'test' : 'content'}"
```

Comment:
This seems like a brittle conditional. At the least I'd recommend putting it in the JS so that there's one place to fix if the tag name changes. For example,
```suggestion
        extraStyleClasses="{isAQuestion() ? 'test' : 'content'}"
```

And in the <script>
```
isAQuestion() {
     return this.state.card.tag.includes('Choice');
}
```